src/lib.rs

```diff
@@ -664,85 +666,84 @@ mod tests {
         assert!(bi.vbe_info_tag().is_some());
         let vbe = bi.vbe_info_tag().unwrap();
         use vbe_info::*;
-        unsafe {
```

Comment:
on Github Diff this looks huge, but it's just:

assert_eq!(vbe.mode, 16762); => assert_eq!({ vbe.mode }, 16762);

to fix a future compile error when reading unaligned fields.

Furthermore the unnecessary `unsafe` block was removed.